### PR TITLE
KOF: Maximum Impact - Maniax database fix

### PR DESCRIPTION
**Pull request by Qwest8K**
Well, I just completed this game and want to make simple correction of database.

### CODE REVIEW

**Comment by RedDevilus** [comment]
You accidentaly added an extra copy on the root and not sure if this PR is even needed for changing - into :

**Comment by MrCK1** [comment]
The PR will have to be squashed to view the changes.

**Comment by Qwest8K** [comment]
When I tried to upload modified database file in the first time, GitHub sent me an error. On the second time, file was loaded normally. I suppose a extra copy happened because of this.

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -33715,8 +33715,9 @@ Name   = King of Fighters 2003, The [SNK Best Collection]
 Region = NTSC-J
 ---------------------------------------------
 Serial = SLPS-25636
-Name   = King of Fighters, The - Maximum Impact
+Name   = King of Fighters, The: Maximum Impact - Maniax
 Region = NTSC-J
+Compat = 5
```
Comment (lightningterror) [inline bin/GameIndex.dbf:33720]: Compatibility is updated through the forum.

**Comment by lightningterror** [comment]
Status?

**Comment by lightningterror** [comment]
@Qwest8K can you update the pr so I can merge this? Otherwise I'd have to close it.

**Pull request closed without merging**